src/utils/match-path.js

```diff
@@ -1,31 +1,27 @@
 import { matchPath } from 'react-router-dom';
-import { routes } from '../config';
+import { params, routes } from '../config';
 
 export const findMatchedPath = (pathname, search) => {
   const matchedFlowchartMainPage = matchPath(pathname + search, {
-    exact: true,
+    exact: false,
```

Comment:
is this changed because of the default query params introduced ?

Reply:
Yes, to match flowchart main page with new query params introduced.